src/Dialogs/Imap/ImapLoginPage.vala

```diff
@@ -107,31 +113,27 @@ public class OnlineAccounts.ImapLoginPage : Gtk.Grid {
         add (display_name_hint_label);
         add (action_area);
 
-        bind_property ("email", email_entry, "text", GLib.BindingFlags.DEFAULT);
+        bind_property ("email", email_entry, "text", BIDIRECTIONAL);
```

Comment:
Do these need to be bidirectional instead of default?

Reply:
Yes because it might get set by loading a configuration when editing an exisiting account and that should be reflected by the entries and the other way it is used to update/create a new configuration and that should use the new entry values. 
Previously it was only bound in one direction and in the signal handlers it was set in the other direction. That's causing an infinite loop in gtk4 though (no idea why not in gtk3 too)